git_machete/client.py

```diff
@@ -968,7 +968,8 @@ def print_line_prefix(branch_: LocalBranchShortName, suffix: str) -> None:
                             fp_branches_formatted: str = " and ".join(
                                 sorted(underline(lb_or_rb) for lb, lb_or_rb in fp_branches_cached[branch]))
                             fp_suffix: str = " %s %s %s seems to be a part of the unique history of %s" % \
-                                             (colored(utils.get_right_arrow(), AnsiEscapeCodes.RED), colored("fork point ???", AnsiEscapeCodes.RED),
```

Comment:
A nuance: here I'd leave `AnsiEscapeCodes` as it used to be. This line is concerned with **rendering**, not **logic**... and hence you're more than justified to operate on `AnsiEscapeCodes`. In fact, using `EdgeColor` doesn't make any sense here